### ISSUE TITLE
BlmapChill: map event make all parts x10 harder

### ISSUE THREAD

**Issue by cyberfig** [DDNetPP/maps#126]
It is too easy for now, and it only covers the police. Please make every part at least x10 harder. Thank you for your attention.

**Comment by ChillerDragon** [comment]
Yeah kais shares your opinion. And I understand it to some degree. But it is just too disrupting. So it will be reverted back to a easier version in [v0182](https://github.com/DDNetPP/maps/pull/127) to stay noob friendly. While it can be refreshing for long time players to see something so well known changed and fun to get challenged by tricky parts I do not think that is a healthy state for a longer time and just cause frustration. It will especially keep away the beginners which potentially make the new generation of active players. I assume it would cause a short peek in pro player activity almost like in a ddrace map release and then a drop in overall server activity. Overall I think "every part at least x10 harder" does not fit a block map. And if something like this were to be done it should probably be a new map.

**Comment by cyberfig** [comment]
it is called "troll police" for a reason, might as well crank up the trollness to maximum in my opinion

**Comment by ChillerDragon** [comment]
Oh you mean for the duration of the event. Oh yea sure we can do another one some times and make it harder yes.
